include/deal.II/matrix_free/face_setup_internal.h

```diff
@@ -770,7 +770,7 @@ namespace internal
 
 
     template <int dim>
-    FaceToCellTopology<1>
+    FaceToCellTopology<1> &&
```

Comment:
I don't think this is appropriate here. You are returning a reference to a temporary.

Reply:
You are right, I was too eager. Fixed now..